test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -86,7 +86,7 @@ def test_pytorch_train_dgl_gpu(pytorch_training, ec2_connection, gpu_only, py3_o
     _, image_framework_version = get_framework_and_version_from_tag(pytorch_training)
     image_cuda_version = get_cuda_version_from_tag(pytorch_training)
     # TODO: Remove when DGL gpu test on ec2 get fixed
-    if Version(image_framework_version) == Version("1.10") and image_cuda_version == "cu113":
+    if Version("1.10") <= Version(image_framework_version) < Version("1.11") and image_cuda_version == "cu113":
```

Comment:
Please change this to `Version(image_framework_version) in SpecifierSet("==1.10.*")`
```suggestion
    if Version(image_framework_version) in SpecifierSet("==1.10.*") and image_cuda_version == "cu113":
```